### PR TITLE
Using requeue cluster instead of resync lease controller

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: Wei Liu <liuweixa@redhat.com>

### CODE REVIEW

**Comment by skeeey** [comment]
refer to https://github.com/open-cluster-management-io/OCM/issues/117

File: pkg/hub/lease/controller.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 
 const leaseDurationTimes = 5
 const leaseName = "managed-cluster-lease"
+const clusterNameLabel = "open-cluster-management.io/cluster-name"
```
Comment (qiujian16) [inline pkg/hub/lease/controller.go:29]: we should put all common lables/annotations in the api repo with good documents. Please create an issue in api repo.

Comment (skeeey) [reply pkg/hub/lease/controller.go:29]: created an issue for this https://github.com/open-cluster-management-io/api/issues/194

File: test/integration/managedcluster_lease_test.go [hunk 2]
```diff
@@ -169,3 +195,15 @@ func assertAvailableCondition(managedClusterName string, status metav1.Condition
 		return nil
 	}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 }
+
+func updateManagedCluster(clusterName string, leaseDuration int32) error {
```
Comment (qiujian16) [inline test/integration/managedcluster_lease_test.go]: nit: updateManagedClusterLeaseDuration

Comment (skeeey) [reply test/integration/managedcluster_lease_test.go]: done

**Pull request merged**